.changeset/late-onions-jog.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': patch
+---
+
+Instead of removing aria-selected attribute for non selected tabs, add aria-selected="false"
```

Comment:
Small suggestion to clarify the changeset (which component this effects) and to use our standard verb form of third-person instead of imperative:

```suggestion
Fixes accessibility by using `aria-selected="false"` for inactive tabs instead of removing `aria-selected="true"` in the tablist of Starlight’s `<Tabs>` component
```